src/client/network.rs

```diff
@@ -371,6 +387,7 @@ impl KafkaConnection {
                id: id,
                host: host.to_owned(),
                stream: stream,
+               closed: false,
```

Comment:
Eh, actually on some more thought, it's probably ok if it turns out that the connection was closed. Further down the chain, it would just return an I/O error I think.